### ISSUE TITLE
Encryption keys brokey again :)

### ISSUE THREAD

**Issue by HiNice2MeetU** [pystardust/ani-cli#646]
**Metadata (please complete the following information)**
Version: 2.0.3
OS: Arch Linux
Shell: Bash
Anime: Tested with,
- Jojo No Kimyou Na Bouken Ougon No Kaze (36)
- Death Note (1)
- Neon Genesis Evangelion (1)
- Jojos Bizarre Adventure (1)

**Describe the bug**
Anime does not stream, comes up with this msg:
base64: invalid input
bad decrypt
140170587273024:error:0606506D:digital envelope routines:EVP_DecryptFinal_ex:wrong final block length:crypto/evp/evp_enc.c:599:
Video URL not found

**Steps To Reproduce**

1. Run `ani-cli Death Note `
2. Choose 1
4. Choose episode 1

**Expected behavior**
Streaming should work

**Comment by iamchokerman** [comment]
yes, gogo changed their keys yet again, they already did it this morning, which we successfully fixed. currently working on a fix for this one

**Comment by HiNice2MeetU** [comment]
ok cool =)

**Comment by Splodienyancat** [comment]
This also affects one piece and by any chance is there a a way to use a more consistent site or maybe find some way to pull the keys from the site in software?

**Comment by hackedXD** [comment]
I'm a newbie to most technical stuff, but how does one go bout getting the encryption keys.

**Comment by iamchokerman** [comment]
tldr: deobfuscate js, locate cryptojs calls, get the key and the iv

**Comment by Shinyzenith** [comment]
how exactly are the keys being retrieved? If anyone can share the steps then I would like to attempt to automate it using CI/CD.

**Comment by amaan211** [comment]
got the same issue.

**Comment by hackedXD** [comment]
> how exactly are the keys being retrieved? If anyone can share the steps then I would like to attempt to automate it using CI/CD.

Exactly. CI integration or like integratetion into a new command. Still idk much bout this project so idk how.

**Comment by theDevAyan** [comment]
same thing is happening when I am trying to download monster

**Comment by DevER-M** [comment]
im getting this problem with death note

**Comment by iamchokerman** [comment]
this issue will happen with all anime.

**Comment by DevER-M** [comment]
oh ok when will this get fixed? @iamchokerman

**Comment by TheGlockMisc** [comment]
I PR'd the fix.

**Comment by Derisis13** [comment]
Fixed in #648

**Comment by pozm** [comment]
They seem to be broken once again for me.
using death note with ep 1.

<img width="651" alt="image" src="https://user-images.githubusercontent.com/44528100/163695679-847334c1-05db-4251-9a52-9aee4bc60822.png">

output
```
Loading episode 1...
Invalid character in input stream.
bad decrypt
8085F30201000000:error:1C80006B:Provider routines:ossl_cipher_generic_block_final:wrong final block length:providers/implementations/ciphers/ciphercommon.c:429:
Video URL not found
```

**Comment by DrKJeff16** [comment]
![image](https://user-images.githubusercontent.com/72052712/163700305-16617b15-dff5-4356-a85e-0b08c8689dd7.png)

Apparently (at least on Windows through Scoop) encryptions just broke again :c
Tested for 3 different animes to no avail.

**Comment by dbackstr** [comment]
Can confirm key change occurred some minutes ago. Can also confirm that I am too dumb to deobfuscate javascript myself, RIP
